src/routes/docs/rgbirdflop/api/layout.tsx

```
@@ -0,0 +1,53 @@
+  }
+  return json.endpoints;
+};
+
+export const useEndpoints = routeLoader$(async () => await getEndpoints());
+
+export const Endpoints = component$(() => {
+  const endpoints = useEndpoints().value;
+  const endpointNames = Object.keys(endpoints) as (keyof typeof endpoints)[];
+
+  return endpointNames.map((path) => <>
+    <h3>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)